### PR TITLE
Minor code cleanups

### PR DESCRIPTION
**Pull request by fmahebert**
A few minor cleanups here and there

### Types of changes:

- [ ] Bugfix
- [ ] New feature
- [ ] Refactor

### Component:

- [ ] Code
- [ ] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests and `clang-tidy`.
  For instructions on how to perform the CI checks locally refer to the [Dev
  guide on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

File: src/NumericalAlgorithms/DiscontinuousGalerkin/NumericalFluxes/Hll.hpp [hunk 1]
```diff
@@ -173,6 +173,13 @@ struct Hll {
         get(max_signal_speed)[s] = std::max(get(max_signal_speed_interior)[s],
                                             -get(min_signal_speed_exterior)[s]);
       }
+      ASSERT(min(get(max_signal_speed) - get(min_signal_speed)) > 0.,
```
Comment (nilsdeppe) [inline src/NumericalAlgorithms/DiscontinuousGalerkin/NumericalFluxes/Hll.hpp]: [nitpick] `0.0` since there's `1.0` below

**Review by nilsdeppe — CHANGES_REQUESTED** [review]
I'm happy for you to squash immediately :)

File: src/NumericalAlgorithms/DiscontinuousGalerkin/NumericalFluxes/Hll.hpp (hunk 1 above)
Comment (fmahebert) [reply src/NumericalAlgorithms/DiscontinuousGalerkin/NumericalFluxes/Hll.hpp]: Oh no! Thanks for catching :)

Comment (fmahebert) [reply src/NumericalAlgorithms/DiscontinuousGalerkin/NumericalFluxes/Hll.hpp]: Done and squashed

**Review by nilsdeppe — APPROVED** [review]

**Review by kidder — APPROVED** [review]

**Pull request merged**